### PR TITLE
MCIM-23: Add existing callerid to numberSelector

### PR DESCRIPTION
**Pull request by Toberal98**
image
image

### CODE REVIEW

File: src/apps/common/submodules/numberSelector/numberSelector.js [hunk 1]
```diff
@@ -228,6 +233,19 @@ define(function(require) {
 							okCallback: addNumberCallback
 						});
 						break;
+					case 'assigned':
+						monster.pub('common.monsterListing.render', {
+							dataList: _.chain(args.numbers.phoneNumbers)
```
Comment (azefiel) [inline src/apps/common/submodules/numberSelector/numberSelector.js:238]: From looking at this [API request](https://github.com/2600hz/monster-ui/blob/master/src/apps/common/submodules/numberSelector/numberSelector.js#L123), the result [looks to return](https://github.com/2600hz/monster-ui/blob/master/src/js/lib/jquery.kazoosdk.js#L319) all active phone numbers in the system, when here we are only interested in phone numbers assigned to an entity.

This state can be inferred by checking whether `used_by` is set for each phone number returned by our API request. Once that's done, don't forget to update `phoneNumbers` to a more meaningful name, such as `assignedNumbers` or something similar, to clarify what this property holds.

File: src/apps/common/submodules/numberSelector/numberSelector.js [hunk 2]
```diff
@@ -30,6 +31,7 @@ define(function(require) {
 				noBuy = _.isBoolean(args.noBuy) ? args.noBuy : false,
 				noExtension = _.isBoolean(args.noExtension) ? args.noExtension : false,
 				noSpare = _.isBoolean(args.noSpare) ? args.noSpare : false,
+				noAssigned = _.isBoolean(args.noSpare) ? args.noSpare : true,
```
[the remark this answers is not in the document]
Comment (Toberal98) [reply src/apps/common/submodules/numberSelector/numberSelector.js:34]: So Sorry i forgot it

File: src/apps/common/submodules/numberSelector/numberSelector.js [hunk 3]
```diff
@@ -123,7 +123,7 @@ define(function(require) {
 						error: next
 					});
 				},
-				listPhoneNumbers = function(next) {
+				listAssignedNumbers = function(next) {
```
Comment (azefiel) [inline src/apps/common/submodules/numberSelector/numberSelector.js:126]: 👍🏻

**Review by azefiel — APPROVED** [review]
Looks good, merge away!

**Pull request merged**